lib/ttnt/tasks.rb

```diff
@@ -0,0 +1,54 @@
+require 'rake/testtask'
+require 'rake/file_list'
+require 'ttnt/testtask'
+require 'ttnt/test_selector'
+require 'rugged'
+
+# Reference: https://github.com/bundler/bundler/blob/master/lib/bundler/gem_helper.rb
+module TTNT
+  class TaskHelper
+    include Rake::DSL if defined? Rake::DSL
+
+    def self.install_tasks
```

Comment:
Ah, you are right! `define` seems better than `install` in my opinion though. :+1: